pkg/dependency/parser/java/pom/parse.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	multierror "github.com/hashicorp/go-multierror"
 	"github.com/samber/lo"
+	"golang.org/x/exp/slices"
```

Comment:
I wanted to point out we should use `slices` and started [adding a new linter](https://github.com/aquasecurity/trivy/pull/6963). Then, I forgot to leave a comment😆 
Fixed in https://github.com/aquasecurity/trivy/pull/6963/commits/2e5cc1f9462ff96615fa9494dcd7d4bbac210067